mix.exs

```diff
@@ -45,6 +45,7 @@ defmodule MongoosePush.Mixfile do
       {:lager, ">= 3.7.0", override: true},
       {:phoenix, "~> 1.4.16"},
       {:open_api_spex, "~> 3.6"},
+      {:mox, "~> 0.5.2"},
```

Comment:
`only: [:test, :integration]`. Also, let's put in closer to other test-only deps.

Reply:
done